bin/boilerplate/test/pass.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Istio Authors. All Rights Reserved.
```

Comment:
was there a decision on `All Rights Reserved` from @smawson ?  The code is inconsistent.

Reply:
Yes， removed